### DISCUSSION TITLE
Getting started with showing pins when clicking your board

### DISCUSSION THREAD

**Discussion by lindseysatterfield** [nss-evening-cohort-14/evening-client-side#163] [Rubber Duck]
So after looking through the README, I noticed under the user stories of Part 2, that one part includes that the user must be able to click on the board and then that will take the user to see all the pins that belong to that board.

**Comment by drteresavasquez** [comment]
To start you need to add a clickable element to your card: https://github.com/drteresavasquez/IN-CLASS-almost-amazon-template/blob/main/src/javascripts/components/authors.js#L15

Then, add a click event that will start the filter: https://github.com/drteresavasquez/IN-CLASS-almost-amazon-template/blob/main/src/javascripts/events/domEvents.js#L77-L83

Then, create a document that allows you to get all the info: https://github.com/drteresavasquez/IN-CLASS-almost-amazon-template/blob/main/src/javascripts/helpers/data/authorBooksData.js

**Comment by drteresavasquez** [comment]
Possible Board => Pins workflow
<img width="1016" alt="Screen Shot 2021-03-08 at 7 49 54 PM" src="https://user-images.githubusercontent.com/29741570/110406214-7d690500-8047-11eb-86e8-0926ac993c5b.png">
